sdk/core/ci.yml

```diff
@@ -73,7 +73,6 @@ stages:
           Value: "DefaultEndpointsProtocol=https;AccountName=notReal;AccountKey=3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333;EndpointSuffix=core.windows.net"
         - Name: PREMIUM_FILE_CONNECTION_STRING
           Value: "DefaultEndpointsProtocol=https;AccountName=notReal;AccountKey=3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333;EndpointSuffix=core.windows.net"
-          Value: "DefaultEndpointsProtocol=https;AccountName=fake;AccountKey=X;EndpointSuffix=core.windows.net"
```

Comment:
Nit: PR title suggestion
Remove incorrect copy of storage TestEnv Value in core CI.yml to fix CI runs

Reply:
Actually, is this causing any errors or failures in CI runs?

Reply:
https://dev.azure.com/azure-sdk/internal/_build?definitionId=1615